lib/child_process.js

```diff
@@ -93,16 +93,16 @@ exports._forkChild = function(fd) {
 };
 
 
-function normalizeExecArgs(command /*, options, callback*/) {
+function normalizeExecArgs(command, options_, callback_) {
   let options;
   let callback;
 
-  if (typeof arguments[1] === 'function') {
+  if (options_ && typeof options_ === 'function') {
```

Comment:
I've though it would eliminate unneeded type check / function call (`isArray()` below) on `undefined` and would be a bit faster. Should I delete them?